src/app/not-found.js

```diff
@@ -1,5 +1,40 @@
+import Link from "next/link";
+{
+  /* the Link tag allows navigation between pages by specifying the path to the desired page  */
+}
+{
+  /* this path is relative to the app folder and is one of files in this folder */
+}
+{
+  /* Next.js preloads the linked page in the background making the next page load almost instantly */
+}
 const NotFound = () => {
-  return <div>404 Page Not Found</div>;
+  return (
+    <div className="flex flex-col items-center justify-center min-h-screen">
+      {/* flex makes all of div a flex container. flex-col stacks all the child elements vertically */}
+      {/* items-center: centers elements horinontally. justify-center: centers elements vertically  */}
+      <h1 className=" leading-none font-mono text-[168px] text-sky-500">
```

Comment:
do NOT use `<h1>` as it messes with SEO
instead use `<p>` tags and use tailwind to increase the size

Reply:
Also, text sizes do not need to be exact, use tailwind classes as is. `text-9xl` may be a good alternative or may be too big, experiment but don't use hardcoded custom values unless absolutely necessacary